python/datasources/acs_population.py

```diff
@@ -587,10 +587,10 @@ def _create_ingesters(self):
 
 def generate_national_dataset_with_all_states(state_df, demographic_breakdown_category):
     all_state_fips = set(state_df[std_col.STATE_FIPS_COL].to_list())
-    return generate_national_dataset(state_df, all_state_fips, demographic_breakdown_category)
+    return GENERATE_NATIONAL_DATASET(state_df, all_state_fips, demographic_breakdown_category)
```

Comment:
why did you decide to capitalize this fn ?

Reply:
Yeah I should change this and come up with a better naming scheme. We will probably have one of these functions for each datasource, or at least every datasource that needs to generate national totals on the fly based on states data quality. 

I capitalized it because after I ran `brew upgrade` pytest was giving me an error saying it wasn't named correctly to be exported. I think it wanted it to start with a capital letter, but I don't really understand why that is the only one it had an issue with.

Reply:
Fixed, IDK what I was doing wrong before. Just using the lower case version and . importing it.